### DISCUSSION TITLE
How to load data from hdfs

### DISCUSSION THREAD

**Discussion by babu1801** [chdb-io/chdb#122] [Q&A] [answered]
When I use query such as:
df = chdb.query("select * from hdfs('hdfs://nn:port/uri_path', 'Parquet')", 'Dataframe')

it throws an excetption:
Code 46: DB::Exception: unknown table function hdfs: or incorrect parameterized view.(UNKNOWN_FUNCTION)

How can I read or write data from/to hdfs file? 

Thank you for any suggestion.

**Comment by auxten** [comment, marked as answer]
The current chdb build did not enable HDFS support. We are planning to have a new release to enable this @nmreadelf

**Reply by babu1801** [reply]
Thanks, I'll make sure to stay informed about any upcoming releases
